# HG changeset patch
# User Wohlstand <admin@wohlnet.ru>
# Date 1530286157 -10800
#      Fri Jun 29 18:29:17 2018 +0300
# Node ID ab02a6b58ca498f1386bf704a1b769b5ebfbbae8
# Parent  58425859a4e522f4d649f1fd885f727d70e5f2e8
Remove "lib" prefix from DLL file on MinGW builds

diff -r 58425859a4e5 -r ab02a6b58ca4 CMakeLists.txt
--- a/CMakeLists.txt	Mon Jun 25 23:00:38 2018 -0400
+++ b/CMakeLists.txt	Fri Jun 29 18:29:17 2018 +0300
@@ -210,6 +210,12 @@
 set(SDL_LIBS "-lSDL2")
 set(SDL_CFLAGS "")
 
+# When compiling shared library on Windows using MinGW
+# avoid the DLL file have a "lib" prefix
+if(WINDOWS)
+  set(CMAKE_SHARED_LIBRARY_PREFIX "")
+endif()
+
 # Emscripten toolchain has a nonempty default value for this, and the checks
 # in this file need to change that, so remember the original value, and
 # restore back to that afterwards. For check_function_exists() to work in
